Servers/using-owncloud-with-dbaas.md

```diff
@@ -0,0 +1,249 @@
+
+### Description
+This CenturyLink Cloud knowledge article provides a walkthrough to install and configure ownCloud on the Linux platform (from scratch and Blueprint), customize ownCloud to utilize CenturyLink Cloud's DBaaS (Beta), SMTP Relay and Object Storage.
+
+For more information, please visit [http://owncloud.org](//owncloud.org)
+
+### Audience
+CenturyLink Cloud Users
+
+### Impact
+After reading this article, the user can configure a customized ownCloud environment to utilize DBaaS, mail relay and Object storage (Enterprise edition) to minimize administration for local resources
+
+
+### Prerequisite
+- Access to the CenturyLink Cloud platform as an authorized user
+- Access to CenturyLink Cloud Database as a Service as an authorized user
+- Access to Object Storage if required, CenturyLink Cloud Object Storage can be used as primary storage for the Enterprise Edition of ownCloud
+
+
+### Postrequisite
+If you want to access your application over the internet, please perform the following tasks after the server is deployed successfully:
+
+1. If you need to connect to your server via the Internet, [Add a Public IP](../Network/how-to-add-public-ip-to-virtual-machine.md) to your server through Control Portal
+
+2. [Allow incoming traffic](../Network/how-to-add-public-ip-to-virtual-machine.md) for desired ports by clicking on the Servers Public IP through Control Portal and configuring appropriately.
+  * The default ports to access the application are: 80, 443
+
+- After adding a public IP for OwnCloud perform the following actions:
+	- Edit the web server configuration files in /etc/apache2/sites-enabled as follows:
+	```
+	{
+    DocumentRoot "/var/www/owncloud/"
+    # Alias /owncloud/ "/var/www/owncloud/"
+    # Alias /owncloud "/var/www/owncloud"
+	}
+    ```
+  - Restart Apache using *sudo  service apache2 restart*
+  - (The following steps will enable management of ownCloud from the public IP address)
+		- Access the server's public IP address using a web browser (with VPN still connected)
+		- ownCloud setup will prompt to add the new IP address as a "trusted domain"
+    ![trusted domain](../images/owncloud/oc-trusted-domain.png)
+		- Click on "Add "IP address" as a trusted domain", it will redirect this request to the private IP address to create the necessary entries to the owncloud configuration file
+    ![add trusted-domain](../images/owncloud/oc-trusted-domain-2.png)
+
+### Procedure
+
+### <a name="installserver"></a> Deploying ownCloud on a New Servers
+**(For Steps using Blueprint, please see [Getting Started with ownCloud Blueprint](../Ecosystem Partners/Marketplace Guides/getting-started-with-owncloud-blueprint.md))**
+Create a Linux server in CenturyLink Cloud using the following knowledge articles:
+- For virtual server, [Create a virtual server](../Servers/how-to-create-customer-specific-os-templates.md)
+- [Create a bare metal server](../../Servers/creating-a-new-bare-metal-server.md) for a private deployment
```

Comment:
you need to fix this.  too many / marks.  (../Servers/creating-a-new-bare-metal-server.md)